src/linpred.jl

```diff
@@ -7,6 +7,14 @@ The effective coefficient vector, `p.scratchbeta`, is evaluated as `p.beta0 .+ f
 and `out` is updated to `p.X * p.scratchbeta`
 """
 function linpred!(out, p::LinPred, f::Real=1.)
+    for i = 1:length(p.delbeta)
+        if isnan(p.delbeta[i])
+            p.delbeta[i] = 0
+        end
+        if isinf(p.delbeta[i])
```

Comment:
```suggestion
        elseif isinf(p.delbeta[i])
```
You could also do `x = p.delbeta[i]` to avoid repeating this, and add `@inbounds` to the loop.